pages/common/htpasswd.md

```diff
@@ -0,0 +1,23 @@
+# htpasswd
+
+> Create and manage htpasswd files to protect web server directories using basic authentication.
+
+- Create/overwrite htpasswd file:
+
+`htpasswd -c {{file_path}} {{user_name}}`
+
+- Add user to htpasswd file or update existing user:
+
+`htpasswd {{file_path}} {{user_name}}`
+
+- Add user to htpasswd file without password verification (for script usage):
+
+`htpasswd -b {{file_path}} {{user_name}} {{password}}`
```

Comment:
Can you alter the description slightly so that its clear what `-b` stands for ? Or maybe use the expanded form of it, if there is one.